install/installer/pkg/components/usage/service.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package usage
+
+import (
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func service(ctx *common.RenderContext) ([]runtime.Object, error) {
```

Comment:
Eventually, we'll also need a network policy to allow server to reach the usage component.

Reply:
Yes, will work on that next.

Reply:
FYI: So far we exclusively have per-component NetworkPolicies, which controll which inboud connections are allowed. So we 1) should already have one for usage (bc `proxy` is able to connect) and 2) it would be a matter of adding 3-5 lines to allow the same for `server`.
Please correct me if I'm wrong :wink:

Reply:
So far nothing talked to usage, so we didn't need a policy. The proxy didn't need to talk to usage.

Reply:
You're right, my bad - mixed with piblic-api. :grimacing: Will go finish my :coffee: before anything else...